DylanTackoor/xenforo-dev-env
# adds comment

marked as a draft until https://github.com/DylanTackoor/xenforo-dev-env/pull/1 is merged, bc I was lazy and didn't split up my branches nicely :)

- Adds `GET|POST /DNPushNotifications` endpoint
- Updates Postman collection

To test:

0. `git checkout feature/threads-with-posts-route`
1. Bring up dev env with `./xenforo.sh`
3. Re-import the Postman collection and fire off all the requests
4. lmk if I should change anything :)


Sample `GET /DNPushNotifications` response:

```json
{
    "success": true,
    "app_on_alert": 0,
    "app_on_conversation": 0,
    "app_on_conversation_message": 0
}
```
